app/javascript/components/import-image-form/import-image-form.schema.js

```diff
@@ -119,6 +119,30 @@ function fieldsForCOS(state, setState, storages, diskTypes, buckets)
     includeEmpty: true,
     clearOnUnmount: true,
     loadOptions: () => buckets,
+    onChange: (value) => {
+      setState({...state, obj_storage_obj_id: value})
+    },
+  },
+
+  {
+    component: componentTypes.SELECT,
+    name: 'obj_storage_obj_id',
+    key: `obj_storage_obj_id-${state['obj_storage_obj_id']}-${state['obj_storage_obj_id']}`,
+    id: 'obj_storage_obj_id',
+    label: __('Choose image object'),
+    isRequired: true,
+    includeEmpty: true,
+    clearOnUnmount: true,
+    loadOptions: () => objects,
+  },
+
+  {
+    component: componentTypes.TEXT_FIELD,
+    name: 'image_name',
+    id: 'image_name',
+    label: __('Custom image name'),
+    isRequired: true,
+    validate: [{ type: validatorTypes.REQUIRED }],
```

Comment:
Also need to prompt user for `osType`. Allowable values: [aix,ibmi,rhel,sles]
https://cloud.ibm.com/apidocs/power-cloud#pcloud-v1-cloudinstances-cosimages-post